src/clj/web/preconstructed.clj

```diff
@@ -0,0 +1,96 @@
+(ns web.preconstructed)
```

Comment:
This is way better than the current solution of pulling from the database. We should put the existing preconstructed decks in here.